packages-web/maps/typings/google.d.ts

```diff
@@ -1,3 +1,3 @@
-interface Window {
+export interface GoogleMapsWindow {
```

Comment:
Can we use `declare window` inside the file where this is needed?

Reply:
Done, did the same with mxGMLocationCache